honeybee_energy/programtype.py

```diff
@@ -0,0 +1,47 @@
+# coding=utf-8
+from __future__ import division
+
+from honeybee.typing import valid_ep_string
+
+
+class ProgramType(object):
+    """Energy Program Type.
+
+    Properties:
+        name
+        people_load
+        lighting_load
+        electric_equipment_load
+        gas_equipment_load
+        infiltration_load
+        ventilation_load
+    """
+
+    def __init__(self, name, people_load=None, lighting_load=None,
+                 electric_equipment_load=None, gas_equipment_load=None,
+                 infiltration_load=None, ventilation_load=None):
+        """Initialize energy program type.
+
+        Args:
+            name: Text string for construction program type. Must be <= 100 characters.
+                Can include spaces but special characters will be stripped out.
```

Comment:
I mean, the whole library really isn't implemented since there's no way that you can create anything simulate-able yet.  I know include WIP objects is usually not good practice but can we just make an exception for this case because 1. this library isn't functional yet 2. the ProgramSet s definitely getting fleshed out in the next PR and 3. I needed a placeholder for the object in order to implement `Room` properties in this PR.

Reply:
Sure. sounds good.